plugins/CoreHome/Columns/UserId.php

```diff
@@ -55,7 +55,11 @@ public function __construct()
      */
     public function onNewVisit(Request $request, Visitor $visitor, $action)
     {
-        return $request->getForcedUserId();
+        $value = $request->getForcedUserId();
```

Comment:
no need to truncate the data in `getForcedUserId()` directly. Should someone use > 200 characters then the visitorId will still be different for different visitors this way (as it does like `substr(sha1($userId), 0, 16)`)